### ISSUE TITLE
[Card] product card text color in dark mode

### ISSUE THREAD

**Issue by ttalbot** [fusionfabric/finastra-design-system#309]
**Describe the bug**
Text color of product card is black on dark, really bad contrast ratio.

**To Reproduce**

1. Go to https://lucid-bassi-bfa8cc.netlify.app/
2. Make sure you're using dark theme, if not toggle the switch on the top right corner
3. Notice the card text on the home page

**Expected behavior**
Text should be white on dark background.

**Screenshots**

![Screenshot 2020-12-18 at 10 11 30](https://user-images.githubusercontent.com/371041/102596520-8ec20580-4119-11eb-9717-b135bd75f58d.png)

**Comment by ArtemSilka** [comment]
Hey @ttalbot , I would like to take on this issue. Let's discuss your expectations!
